spec/controllers/bookmarks_controller_spec.rb

```diff
@@ -10,7 +10,11 @@ def it_redirects_to_user_login
 
   describe "new" do
     context "without javascript" do
-      it "redirects anyone not logged in" do
+      let(:chaptered_work) { create(:work) }
+      let(:chapter2) { create(:chapter, work: chaptered_work) }
+      let(:bookmark) { create(:bookmark, bookmarkable_id: chaptered_work.id) }
+
+      it "redirects and should not return the form for anyone not logged in" do
```

Comment:
I think we can just use `it "redirects logged out users" do` here, since we're not explicitly checking the form isn't rendered. If we wanted more specificity, we could say `it "redirects logged out users to the user login page" do`

Reply:
Agreed